internal/handlers/msg_serverstatus.go

```diff
@@ -16,20 +16,70 @@ package handlers
 
 import (
 	"context"
+	"os"
+	"path/filepath"
+	"time"
 
 	"github.com/FerretDB/FerretDB/internal/types"
 	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+	"github.com/FerretDB/FerretDB/internal/util/must"
 	"github.com/FerretDB/FerretDB/internal/wire"
 )
 
 // MsgServerStatus OpMsg used to get a server status.
 func (h *Handler) MsgServerStatus(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
+	document, err := msg.Document()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	m := document.Map()
+	db, ok := m["$db"].(string)
+	if !ok {
+		return nil, lazyerrors.New("no db")
+	}
+
+	host, err := os.Hostname()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+	exec, err := os.Executable()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	uptimeMillis := time.Now().UnixMilli() - h.startUnixTime
+	uptimeSecs := uptimeMillis / 1_000
```

Comment:
With comment above, that will be just `time.Since(h.startTime).Seconds()`

Reply:
Agreed. Thanks!